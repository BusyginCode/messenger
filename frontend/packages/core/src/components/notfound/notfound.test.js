import React from 'react'
import { shallowWithIntl } from '@messenger/testing'
import NotFound from './notfound'

describe('app/components/NotFound', () => {
  it('should render correctly', () => {
    const staticContext = {}
    expect(shallowWithIntl(<NotFound staticContext={staticContext} />).dive()).toMatchSnapshot()
    expect(staticContext.is404).toBe(true)
  })
})
