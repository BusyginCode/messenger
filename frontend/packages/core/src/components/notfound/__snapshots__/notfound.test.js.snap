// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/components/NotFound should render correctly 1`] = `
<Box
  align="center"
  className="notFound"
  component="div"
  justify="center"
>
  <Box
    className="notFoundImage"
    component="div"
  />
  <Box
    component="div"
    direction="column"
    justify="center"
    left="l"
  >
    <AText
      breed="title"
      color="alfa-text-body"
      size="xl"
    >
      404 error
    </AText>
    <Box
      bottom="l"
      className="description"
      component="div"
      inject={true}
      top="m"
    >
      <AText
        align="left"
        breed="body"
        color="alfa-text-body"
        size="l"
      >
        We can't seem to find the page you're looking for
      </AText>
    </Box>
    <Button
      actionType="navigational"
      circle={false}
      component={[Function]}
      intent="primary"
      rounded={false}
      size="l"
      to="/"
    >
      GO TO HOMEPAGE
    </Button>
  </Box>
</Box>
`;
